src/main/scala/fastq.scala

```diff
@@ -98,22 +132,60 @@ case object fastq {
     )
     : FASTA := ((fasta.header.type := fasta.header.Raw) :: (fasta.sequence.type := fasta.sequence.Raw) :: *[AnyDenotation]) =
       FASTA(
-        fasta.header( me.getV(id) toFastaHeader )                  ::
-        fasta.sequence( FastaSequence((me getV sequence).asString) )  :: *[AnyDenotation]
+        fasta.header( me.getV(id).toFastaHeader )                   ::
+        fasta.sequence( FastaSequence(me.getV(sequence).asString) ) :: *[AnyDenotation]
       )
 
-    def toLines(implicit
-      getId: AnyApp1At[findS[AnyDenotation.Of[id.type]], RV] { type Y = id.type := id.Raw },
-      getSeq: AnyApp1At[findS[AnyDenotation.Of[sequence.type]], RV] { type Y = sequence.type := sequence.Raw },
-      getPlus: AnyApp1At[findS[AnyDenotation.Of[plus.type]],RV] { type Y = plus.type := plus.Raw },
-      getQual: AnyApp1At[findS[AnyDenotation.Of[quality.type]],RV] { type Y = quality.type := quality.Raw }
-    )
-    : Seq[String] =
-      Seq(
-        (me getV id)       .asString,
-        (me getV sequence) .asString,
-        (me getV plus)     .asString,
-        (me getV quality)  .asString
-      )
+    def asString: String =
+      (
+        seq.head.value.asString                 ::
+        seq.tail.head.value.asString            ::
+        seq.tail.tail.head.value.asString       ::
+        seq.tail.tail.tail.head.value.asString  ::
+        Nil
```

Comment:
See #11 If we fix the Raw type, we can use `getV` here without ~~fear~~implicits :wink: :ok_hand: